pom.xml

```diff
@@ -336,9 +336,31 @@
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.8.1</version>
           <configuration>
+            <compilerId>javac-with-errorprone</compilerId>
+            <forceJavacCompilerUse>true</forceJavacCompilerUse>
             <source>1.8</source>
             <target>1.8</target>
+            <compilerArgs combine.children="override">
+              <compilerArg>-XepAllErrorsAsWarnings</compilerArg>
```

Comment:
This is temporary to avoid breaking the build, but eventually we should remove it.